mne/io/open.py

```diff
@@ -255,7 +257,6 @@ def _show_tree(fid, tree, indent, level, read_limit, max_str):
                 out += [next_idt + prepend + str(k) + ' = ' +
                         '/'.join(this_type) + ' (' + str(size) + ')' +
                         postpend]
-                out[-1] = out[-1].replace('\n', u'¶')
```

Comment:
I believe it was unicode before. I think optimally this would have to check all the possible platforms to work 100% of the time. Not worth the effort IMO.